bindings/java/CMakeLists.txt

```diff
@@ -74,21 +101,27 @@ add_jar(yarp_java_matlab_jar
         SOURCES ${JAVA_MATLAB_UTILS_SRCS}
         INCLUDE_JARS yarp_java_jar
         OUTPUT_NAME yarp_matlab_java
-        VERSION 1.0)
+        VERSION ${YARP_VERSION})
 
 add_dependencies(yarp_java_jar ${SWIG_MODULE_yarp_java_REAL_NAME})
 
 # installation part
 install(TARGETS ${SWIG_MODULE_yarp_java_REAL_NAME}
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/jni
+        DESTINATION ${CMAKE_INSTALL_JNIDIR}
         COMPONENT bindings_java)
 
 install_jar(yarp_java_jar
             DESTINATION ${CMAKE_INSTALL_DATADIR}/yarp/java
             COMPONENT bindings_java)
+install_jni_symlink(yarp_java_jar
```

Comment:
What does this command do?

Reply:
It installs symbolic link to the generated jar in a different directory (in this case in share/java).